snap/hooks/post-refresh

```diff
@@ -14,6 +14,15 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+if [ -d $SNAP_COMMON/host-hunspell ]; then
+    # Clean up old Firefox snap mount for host hunspell; we now use
+    # the system-files interface for host hunspell dictionaries.
+    # See the 'disconnect-plug-host-hunspell' hook for more details.
+    snapctl umount $SNAP_COMMON/host-hunspell || true
+    rm -r $SNAP_COMMON/host-hunspell || true
+    exit 0
+fi
+
```

Comment:
We should leave that change out, referring to the corresponding commit message from #292d7309

```
This is a temporary measure likely needed only for the users of 'nightly' and 'beta' who had their Firefox snap updated
```

In fact we could probably also remove it from beta and nightly at this point since it was an transitional upgrade issue and the non optional refresh for snap probably means the users who had the previous version went through the change already